examples/equity-token/contracts/modules/EquityERC20TransferLimitsModule.sol

```diff
@@ -14,6 +14,6 @@ contract EquityERC20TransferLimitsModule is ERC20TransferLimitsModule {
         IAssetF.Context memory ctx_;
```

Comment:
IDK, maybe rename this function to `getContextKeyBySelector()` to avoid naming collision?